test/new/db/cmd/cmd_pd

```diff
@@ -1811,7 +1811,7 @@ pd 1
 e scr.color=3
 ec linehl none
 pd 1
-ec linehl #002f00
+"ec linehl #002f00"
```

Comment:
`#whatever` is considered a comment. `#` is a special char, so in the new parser you can easily escape it `\#`, but for now let's just quote the entire command.

Reply:
i dont see why we should use # in here. the syntax should be rgb:002f00

Reply:
this should be fixed in r2, the test is wrong

Reply:
I agree but I don't want to touch too much existing commands as they need to be changed anyway in the future to accept argc,argv

Reply:
then we must fix this in master and u should rebase

Reply:
I have changed the test to use `rgb:`